spec/components/schemas.yaml

```diff
@@ -1073,7 +1077,9 @@ components:
         end_datetime:
           $ref: "#/components/schemas/datetime"
         expires:
-          $ref: "#/components/schemas/datetime"
+          readOnly: true
```

Comment:
`expires` is should actually be writable. It can optionally be set to delete the data in the future.

https://jira.swisstopo.ch/browse/PB-847

Reply:
Alright, will adapt. Thanks!

This answers my question from the PR description:
> ❓ With this fix, the following fields in "properties" are still read-only, so hidden in updateFeature: created, updated and expires. This is in line with what we see in [v0.9](https://sys-data.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#tag/Data-Management/operation/putFeature). For created and updated, this sounds reasonable: They are set automatically and should not be defined in a PUT request. But for expires (description: "Date time when this presigned URL expires and is not valid") this might be a bug as well: In the DRF validator [the field is examined](https://github.com/geoadmin/service-stac/blob/develop/app/stac_api/validators.py#L428), so I suppose it is writeable. Should the OpenAPI specs be fixed here too?